guides/change-management.md

```diff
@@ -0,0 +1,31 @@
+# Change management
+
+New projects and features often have questions about browser support: which browsers and features need to be supported, when it is acceptable to provide alternate displays, etc.  
+
+[Browser support standards](https://github.com/cfpb/development/blob/master/guides/browser-support.md) have been established by the Browser Support Working Group. In some cases, however, there will be reasons that customers, product owners, or engineering will want to change what they are required to support in their application. This could be because a new leading-edge technology delivers value that cannot be achieved otherwise, or the customer's target market is not our normal user base and will use a known set of browsers, etc.
+
+Without a clear idea of the process for requesting these exceptions, projects will not be able to effectively plan their engineering efforts. This document outlines a decision making process for requesting an exception to current browser support standards.
+
+## Process flow
+
+1. Project starts and team assesses front end needs
+2. If project team decides their needs are unique and cannot be supported by current CFPB browser standards, they create a Hubcap GHE Issue to review the situation, including answers to the following questions:  
+    - What is the device/OS/browser/feature for which you would like to drop support in this app?
+    - What problem does dropping this support solve?
+        - Are you dropping support because engineering compatible solutions is too hard, or because no compatible solution yet exists? If the former, discuss the business impact of adding the time to build the compatible solution.
+    - What is your fallback for providing this content in an accessible way to unsupported browsers?
+    - What percentage of site users will not have access to your app because of this?
+        - Request information from Digital Analytics so team can assess based on exact numbers. Include audiences if relevant.
+        - Include affected pages from [list of key pages]()
```

Comment:
Need to include a link here -- maybe to the browser support levels table, which lists key pages

Reply:
I think this is the canonical location of the list: https://github.cfpb.gov/CFGOV/platform/wiki/Key-pages-for-QA%2C-testing%2C-analytics

Reply:
Updated in https://github.com/cfpb/development/pull/165/commits/ecb097c4ce1d64a4bbcedcd537d2f783932cb11d -- thanks!